app/src/main/java/eu/kanade/tachiyomi/data/track/bangumi/dto/BGMSearch.kt

```diff
@@ -25,9 +26,9 @@ data class BGMSearchItem(
 ) {
     fun toTrackSearch(trackId: Long): TrackSearch = TrackSearch.create(trackId).apply {
         remote_id = this@BGMSearchItem.id
-        title = nameCn
+        title = nameCn.ifEmpty { name }
         cover_url = images?.common ?: ""
-        summary = this@BGMSearchItem.name
+        summary = name + this@BGMSearchItem.summary?.let { "\n$it" }.orEmpty()
```

Comment:
don't show the name?

Reply:
> don't show the name?

But the "name" field is the JP name. The search entry title is normally the Chinese name unless that doesn't exist, then it falls back to the JP name.
We've previously shown the JP name in the summary, thought it would be convenient for users to be able to see it still.

Reply:
Should probably have it as 
```
Native title: $name

$summary
```
Native title being in summary. Also maybe don't include it in summary if Chinese name is blank?

Reply:
Should it say "native title" or is there a was to make it a localised string? Or maybe put it in Chinese? 😅

Reply:
It's a Chinese tracker. Hard coding Chinese is fine